README.md

```diff
@@ -30,76 +41,90 @@ $ helm install ohmyglb ohmyglb/ohmyglb
 See [values.yaml](https://github.com/AbsaOSS/ohmyglb/blob/master/chart/ohmyglb/values.yaml)
 for customization options.
 
-### Local Playground
-
-####  Deploy local cluster
-
-```sh
-$ make deploy-local-cluster
+### Local Playground Install
+
+#### Environment prerequisites
+
+ - install **GO 1.14**
+ 
+ - install **GIT**     
+    - ensure SSH key is set for your account; see https://github.com/settings/keys  
+    
+ - install **gnu-sed** if you don't have it  
+    - update $PATH in `~/.bash_profile` by sed path `export PATH=/usr/local/opt/gnu-sed/libexec/gnubin:$PATH` 
```

Comment:
It's a mac only recommendation, please explicitly mention it
Also probably snippet of 
```
$ brew install gnu-sed
```
will be handy